08.ls_object/short_formatter.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class ShortFormatter
+  COLUMN = 3
+
+  def initialize(files)
+    @files = files
+  end
+
+  def output_file
+    max_length = @files.max_by(&:length).length
+
+    divide_files_by_column(@files, COLUMN).each do |files|
+      files.each do |file|
+        print format("%-#{max_length + 1}s", file)
+      end
+      puts
+    end
+  end
+
+  private
+
+  def divide_files_by_column(files, column)
+    number_of_directory_division = (files.size.to_f / column).ceil
+    divided_directories = files.each_slice(number_of_directory_division).to_a
+
+    difference_number_of_elements = divided_directories.first.size - divided_directories.last.size
```

Comment:
```suggestion
    diff_count = divided_directories.first.size - divided_directories.last.size
```

寿命が短いのでこれぐらいの名前でもいいかもしれません。

あと、変数名ってあまりofを使わないんですよね。
b_of_aと書くより、a_bと書くことが多い気がします。

Reply:
`diff_count`に命名変更しました。